enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -60,424 +55,481 @@ recommended to review it before proceeding.
+1. The updated policy applies to all clusters in the `stage` group. These clusters have the `rolloutStatus` of
+   `InProgress`. The root policy `status.rolloutStatus` value is `InProgress`.
+1. All the clusters in the `stage` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The user sets the `spec.decisionGroups[2].rolloutApproved` (`prod` decision group) value to `true`.
+1. The updated policy applies to all clusters in the `prod` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `prod` group become compliant. Their `rolloutStatus` is set to `Succeeded`.
+1. The user sets the `spec.decisionGroups[0].rolloutApproved` (`stage` decision group) value to `true`.
+1. The updated policy applies to all clusters in the `stage` group. These clusters have the `rolloutStatus` of
+   `InProgress`.
+1. All the clusters in the `stage` group become compliant. Their `rolloutStatus` is set to `Succeeded`. The root policy
+   `status.rolloutStatus` value is `Succeeded`.
+
+#### Common Situations
+
+##### A New Cluster Is Added During a Rollout
+
+When a new cluster is added, it receives the same policy as the others in the same group currently have. For example, if
+there is a policy update and a cluster is added to the `stage` group while the `dev` group is still `InProgress`, the
+new cluster would receive the last successful policy version, if any, until the rollout progressed to the `stage` group.
+
+If the cluster is added to an earlier group than the group that is `InProgress`, the rollout switches back to that group
+and waits for the cluster to rollout before resuming progress at the point previous to the new cluster being added.
+
+##### A New Cluster Is Added After a Rollout
+
+When a managed cluster is added to the placement after the rollout completed, the rollout does not restart. The cluster
+directly receives the last successful policy definition and has a rollout status of `NotApplicable`.
+
+##### A Cluster In The Group Is Offline
+
+If a cluster does not provide any compliance after the `progressDeadline` value has elapsed, the cluster is set to
+`TimeOut` and the rollout fails. Assuming no additional rollout is started, when the cluster comes back online, it will
+receieve the last successful policy version, if any.
+
+##### Retrying a Failed Rollout
+
+This is only necessary if `progressDeadline` is used since this is the only way a rollout can fail. Otherwise, the
+rollout indefinitely waits for the clusters to become compliant.
+
+In this situation, the first option is to update the policy spec in some way which would completly restart the rollout.
+
+The second option is to use the `spec.retryRollout.rolloutUID` field on the `Rollout` object. When set to a UID that
+matches `status.rolloutUID`, the Governance Policy Propagator will restart the rollout and create a new rollout UID.
```

Comment:
Are we going to save snaps of policy + rollout?

Reply:
Could you please rephrase the question?